packages/ui/jest.config.ts

```diff
@@ -8,8 +8,10 @@ const config: Config = {
     '!<rootDir>/src/index.ts',
     // ignore internal `debugUtils` from coverage thresholds
     '!<rootDir>/**/debugUtils.ts',
-    // ignore coverage for style-dictionary type declaration file
-    '!<rootDir>/src/theme/types/style-dictionary.d.ts',
+    // ignore coverage for copied style-dictionary functions
+    '!<rootDir>/src/theme/createTheme/resolveObject.ts',
+    '!<rootDir>/src/utils/references.ts',
+    '!<rootDir>/src/utils/groupMessages.ts',
```

Comment:
Why would we ignore testing this code? By migrating it in to our code base we are now liable for it